packages/api/src/modules/user/user.service.ts

```diff
@@ -273,31 +266,21 @@ export class UserService {
         return email;
     }
 
-    async addRolesToUser(
-        user: UserDto | string,
-        roles: RoleEnum[]
-    ): Promise<{ success: true; user: UserDto } | UserServiceError> {
+    async addRolesToUser(user: UserDto | string, roles: RoleEnum[]): Promise<{ success: true; user: UserDto }> {
         if (typeof user === "string") {
             const foundUser = await userRepository.findByEmail(user);
             if (!foundUser) {
-                return {
-                    success: false,
-                    message: "User email does not correspond to a user",
-                    code: UserServiceErrors.USER_NOT_FOUND
-                };
+                throw new EntityNotFoundError("User Not Found", UserServiceErrors.USER_NOT_FOUND);
```

Comment:
Pourquoi on change le message d'erreur ?

Reply:
Je voulais homogénéisé avec des messages moins verbeux

Reply:
:thinking: J'ai l'impression que ça se réfléchit, ça me paraît pas évident qu'il faudrait aller au + succinct

Reply:
Je dirais que la règle, c'est tant qu'on peut comprendre l'erreur sans avoir besoin du contexte (la route appelé par exemple) c'est ok ! User not found ne me choque pas plus que ça mais c'est vrai qu'on perd de l'info ! Après est-ce que l'info est vraiment importante ? Je dirait que oui si je devrais implémenter cette api dans mon front alors que je n'ais pas la main sur le code du back